schema/me/conversation.js

```diff
@@ -23,9 +143,13 @@ export default {
         client_application_id: IMPULSE_APPLICATION_ID,
       })
       .then(data => {
-        return impulse.with(data.token, { method: "GET" })(`conversations/${id}`).then(impulseData => {
-          return impulseData
-        })
+        return impulse
+          .with(data.token, { method: "GET" })(`conversations/${id}`, {
+            expand: ["messages"],
```

Comment:
@l2succes do you know of a way to pass this as a param only when we're querying for `messages`? I guess it's not a huge deal if it's there on every query but idk 🤷‍♀️